docs/validator/validator-faq.md

```diff
@@ -23,62 +19,70 @@ More specific steps:
 3. Send a staking transaction using your wallet / CLI with your account including amount and public key from newly generated key pair.
 4. Wait until the node becomes a validator
 
-## What is 'staking'?
+### What is 'staking'?
 
-We call staking a process of sending `StakeTransaction` that inform the network that given account wants to become a validator in upcoming epochs. This particular type of transaction must provide a public key and staking amount.
-After the transaction is sent, a node that has a private key associated with the public key in the staking transaction must wait until two epochs to become validator. **important**: a node can become a validator only if the amount in the staking transaction is above the seat price defined by the protocol.
+We call staking a process of sending `StakeTransaction` that informs the network that a given account wants to become a validator in upcoming epochs. This particular type of transaction must provide a public key and staking amount. After the transaction is sent, a node that has a private key associated with the public key in the staking transaction must wait until two epochs to become a validator. 
+**important**: a node can become a validator only if the amount in the staking transaction is above the seat price defined by the protocol.
 
-## What is a minimum amount to stake as a validator?
+### What is an Epoch?
+
+An epoch is the interval of time that consists of a fixed number of consensus rounds. Currently, one epoch lasts for about the duration of half a day and is used to
```

Comment:
there is no guarantee of "fixed number of consensus rounds"

Reply:
That's about what Alex said i.e. that an epoch is not a time duration but a number of consensus rounds. Let me elaborate that there is no guarantee